src/main/request.ts

```diff
@@ -135,6 +135,25 @@ export class Request {
         });
     }
 
+    async getFile(url: string, type: string, options: RequestOptions = {}) {
+        const { body, query, headers } = options;
+        const res = await this.send('get', url, {
+            headers: {
+                'content-type': type,
```

Comment:
Why not just leave the `content-type` as a part of the `headers`?

Reply:
well, theoretically one could override type with headers, but this way with type argument it should give better developer experience.